docs/register/prop-method-injection.rst

```diff
@@ -43,12 +154,45 @@ If the component is a :ref:`reflection component <register-registration-reflecti
     // is important!
     builder.RegisterType<C>().PropertiesAutowired(new MyCustomPropSelector());
 
+.. note::
+
+  Properties set on a component because they are ``required`` may lead to duplicate injection of a property if ``PropertiesAutowired`` is also used on that component.  Consider using ``PropertiesAutowired(PropertyWiringOptions.PreserveSetValues)`` to avoid repeating property injection.
+
+Manually Specifying Properties
+------------------------------
+
 If you have one specific property and value to wire up, you can use the ``WithProperty()`` modifier:
 
 .. sourcecode:: csharp
 
     builder.RegisterType<A>().WithProperty("PropertyName", propertyValue);
 
+Overriding Required Properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
------------------------------
```

I think this would basically end up creating a "bullet list of one thing" underneath "Manually Specifying Properties." Can't have a bullet list of one.